src/agent/rustjail/src/container.rs

```diff
@@ -1655,7 +1660,7 @@ mod tests {
             &slog_scope::logger(),
             &Hook {
                 path: sleep,
-                args: vec!["2".to_string()],
+                args: vec!["sleep".to_string(), "2".to_string()],
```

Comment:
Looking at the existing unit tests, I don't see one that correctly runs a hook and checks the hook output (the `xargs` test doesn't really count imho). That probably explains why this feature has been broken too! ;)

Therefore, to prove the fix on this PR works please can you add a new test that does the equivalent of something like this:

```bash
tmpdir=$(mktemp -d)
touch ${tmpdir}/a ${tmpdir}/b ${tmpdir}/c
```

The test should then check to ensure all three files are created.

Reply:
Thanks for your advice, I will do some fixes about tests when I figured out other problems I found related to using hook in guest vm.